ui/webui/src/components/HelpDrawer.jsx

```diff
@@ -25,15 +24,9 @@ import {
     DrawerContentBody,
     DrawerHead,
     DrawerPanelContent,
-    Text,
-    TextContent,
-    TextVariants,
-    Title
 } from "@patternfly/react-core";
 
-const _ = cockpit.gettext;
-
-export const HelpDrawer = ({ isExpanded, setIsExpanded, children }) => {
+export const HelpDrawer = ({ isExpanded, setIsExpanded, helpContent, children }) => {
```

Comment:
It is probably not related to change in this PR, but is there a reason for passing the setIsExpanded setter to the HelpDrawer?

Reply:
The setter is called with `false` when you click the `X` button inside of the drawer. I think that makes sense, Setting isExpanded directly wouldn't cause rerender.

Reply:
Oh sure, I was just blind, it makes sense but I couldn't find it in the diff.
Sorry for the noise...